core/src/main/resources/lib/hudson/project/configurable/adjunct.js

```diff
@@ -0,0 +1,12 @@
+window["lib_hudson_project_configurable_build_now_callback"] = function (
+  el,
+  ev
+) {
+  let parameterized = el.dataset.parameterized;
+  let success = el.dataset.success;
+  if (parameterized === "false") {
```

Comment:
The behavior when `it.parameterized` is so different that it would probably be clearer to just do that switch on the server side using `j:choose`.

Reply:
I had it like that for a while, but didn't like the resulting duplication of `l:task`.

(In favor of this would be the simpler JS, and the removal of the `div` holding `data-*` attributes in the parameterized case.)